src/views/TechnicalAssistance/RequestForm/Attendees.tsx

```diff
@@ -21,7 +32,36 @@ import { trbRequesterSchema } from 'validations/trbRequestSchema';
 import { AttendeeFields, AttendeesTable } from './AttendeesForm/components';
 import AttendeesForm from './AttendeesForm';
 import Pager from './Pager';
-import { FormStepComponentProps, StepSubmit } from '.';
+import { FormStepComponentProps, StepSubmit, TrbFormAlert } from '.';
+
+// Make FormStepComponentProps conditionally required on the presence of fromTaskList
```

Comment:
It looks like `FormStepComponentProps` is only required if `fromTaskList` is specifically set to false, but the props are still optional if `fromTaskList` is undefined.

Updating `fromTaskList` in the first part of the union to `{fromTaskList: true}` fixes it.